.github/workflows/tests-off.yml

```
@@ -0,0 +1,58 @@
+    runs-on: ubuntu-22.04
+
+    env:
+      BUILD_TYPE: ${{matrix.BUILD_TYPE}}
+      STANDALONE: ${{matrix.STANDALONE}}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - BUILD_TYPE: Release
+            STANDALONE: ON
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)